src/envo/envo-edit.owl

```diff
@@ -40171,6 +39843,535 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3400002> "ar
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_3510000> "Wood dust is typically generated through processes where wood is chipped, sawed, turned, drilled or sanded. The chemical composition of wood dust depends on the species of tree. Wood dust is not typically produced for specific uses. However, it is used commercially for wood composts. The primary routes of potential human exposure to wood dust are inhalation and dermal contact. It is known to be a human carcinogen.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510000> "wood dust"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510000> <http://purl.obolibrary.org/obo/ENVO_00002008>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510001> (brake fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://doi.org/10.1520/JTE14254") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Brake_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510001> "A hydraulic fluid which is 1) used in hydraulic brake and hydraulic clutch applications in vehicles to amplify braking force and 2) composed primarily of ethylene glycol, polyglycols, silicone fluids, and isobutyl alcohol.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510001> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510001> "2020-08-03T17:32:09Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510001> "brake fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510001> <http://purl.obolibrary.org/obo/ENVO_3510004>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510002> (automatic transmission fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Automatic_transmission_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510002> "Fluid used for valve operation, brake band friction, and gear lubrication within an automatic vehicle transmission.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510002> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510002> "2020-08-03T17:34:16Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510002> "ATF")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510002> "automatic transmission fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510002> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510004> (hydraulic fluid)
```

Comment:
This, and several other classes, are asserted subclasses of [manufactured product](http://purl.obolibrary.org/obo/ENVO_00003074). While not wrong, this should be done by inference [1] and the asserted subclass should be somewhere in the environmental material hierarchy. 

I'd also add axioms like `'has quality' some 'quality of a liquid'` so that it gets reasoned into the `liquid environmental material` hierarchy, unless you assert it to be there.

[1] After moving this to a good spot in the material hierarchy, add `'output of' some 'manufacturing process'` to the subclass axioms and it will be reasoned into the manufactured product hierarchy.

Reply:
Ok I wont really know the best place for these terms. Can someone help us?

dye
India ink
fountain pen ink
varnish
vitreous enamel
turpentine
naphtha
automatic transmission fluid
hydraulic fluid
motor oil

Where in the ME hierarchy should i stick these?

Reply:
@matentzn this is typically why we ask people in the [merge workflow](https://github.com/EnvironmentOntology/envo/wiki/ENVO-Robot-template-and-merge-workflow) to follow our template and share it with us so that we can review and give suggestions prior to trying to merge it in. 

My suggested superclass for those terms would be `manufactured product` http://purl.obolibrary.org/obo/ENVO_00003074

Reply:
But I see you've already merged many terms in 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/12255688/107641555-10469780-6c74-11eb-89d9-9aa4c34c7fd3.png">

This must be what @pbuttigieg was referring to. I think this merge was done a little hastily without giving a chance to make sure they conform to our standards.  

<img width="674" alt="image" src="https://user-images.githubusercontent.com/12255688/107641722-3d934580-6c74-11eb-8788-e98e48b9fbd2.png">

note the definition could use a little reformatting see https://github.com/EnvironmentOntology/envo/wiki/ENVO-Robot-template-and-merge-workflow#definitions for the instructions about envo's standards. 

also the xref to air should be on the comment and or definition. Following our template would help sort these things out hence why we made it.